### PR TITLE
Modified to use tags instead of node_ids in flows

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/commons#603
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
Need to be verified

File: tendrl/integrations/ceph/flows/add_osds/__init__.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def run(self):
             new_params['Node[]'] = [node_id]
             new_params['ssh_setup_script'] = ssh_setup_script
             payload = {
-                "node_ids": [node_id],
+                "tags": ["tendrl/node_%s" % node],
```
Comment (r0h4n) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: it is "node_id" and not "node"

Comment (shtripat) [inline tendrl/integrations/ceph/flows/add_osds/__init__.py]: ah yes. will change that

**Comment by shtripat** [comment]
Verified the changes with add osd flow and works as expected.

**Pull request merged**